packages/amplify-cli/src/extensions/amplify-helpers/apply-auth-mode.ts

```diff
@@ -0,0 +1,83 @@
+import { $TSContext } from "amplify-cli-core";
+import { getProjectMeta } from './get-project-meta';
+
+export function isValidGraphQLAuthError(message: string): boolean {
+  return (
+    message === `@auth directive with 'iam' provider found, but the project has no IAM authentication provider configured.` ||
```

Comment:
Can you put these error messages into constants at the top of this file. They are pretty long and we're checking them in multiple places.

Reply:
Updated.